code/modules/lighting/light_source.dm

```diff
@@ -169,6 +170,11 @@
 					round(lum_b * ., LIGHTING_ROUND_VALUE)
 				)
 
+			else
+				effect_r += 0
+				effect_b += 0
+				effect_g += 0
```

Comment:
it takes 5 seconds to figure out that's a list, scroll up to the top and see effect_r is a list